recipes/catch2/2.x.x/conanfile.py

```diff
@@ -49,7 +49,13 @@ def _configure_cmake(self):
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            print("aplying patch %s" % (patch,))
```

Comment:
```suggestion
```

Not needed.

When you really need to print something, use `self.output.info` instead.